en-US/Players_Guide/05_Spellcasting/Spellcasting.md

```diff
@@ -1,6 +1,301 @@
+
+Some spells require an attack roll to determine if the spell effect hits the target.
+Your spell attack bonus equals your spellcasting ability modifier plus your proficiency bonus.
+Most spell attack rolls involve ranged attacks.
+You have disadvantage on a ranged attack roll if you’re within 5 feet of a hostile creature that can locate you and isn’t incapacitated.
+
+#### The Schools of Magic <!-- In the WotC SRD, this is an info box. -->
+
+Academies of magic group spells into eight schools of magic, believing that all magic functions similarly, regardless of its source.
+Scholars, especially Wizards, apply these categories to all spells.
+
+The schools of magic help describe spells; they have no rules of their own, although some rules refer to the schools.
+
+**Abjuration** spells protect, some with aggressive uses.
+They create barriers, negate harm, harm trespassers, or banish creatures.
+
+**Conjuration** spells transport objects and creatures.
+Some summon or teleport, while others create objects.
+
+**Divination** spells reveal secrets, futures, hidden locations, illusions’ truths, or distant visions.
+
+**Enchantment** spells affect minds, influencing behavior.
+They can make enemies friends, force actions, or control creatures.
+
+**Evocation** spells manipulate magical energy, producing desired effects like fire or lightning blasts or healing.
+
+**Illusion** spells deceive senses or minds, causing phantom sights, sounds, or memories.
+Some create phantom images, while others plant images directly in minds.
+
+**Necromancy** spells manipulate life and death energies.
+They grant life force reserves, drain life, create undead, or resurrect the dead.
+Creating undead through Animate Dead is evil and rarely used.
+
+**Transmutation** spells change properties, turning enemies into harmless creatures, bolstering allies, making objects move at will, or enhancing healing.
+
+#### Combining Magical Effects
+
+The effects of different spells combine while their durations overlap.
+However, the most potent effect from multiple castings applies while their durations overlap.
+For instance, if two Clerics cast Bless on the same target, the character gains the spell’s benefit only once, not twice.
+
 ### Spell Lists
+> **Spell Name Changes**
+>
+> Some spell names in Free5e have been renamed, in most cases to honor those who supported the project financially.
+> If a product (e.g. an adventure or a monster stat block) mentiones one of these spells, you can use the renamed version instead.
+> - Acid Arrow → Bragolbeleg's Acid Bolt
+> - Arcane Hand → Morscheck‘s Hand
+> - Black Tentacles → Coreador's Black Tentacles
+> - Druidcraft → Naturecraft
+> - Floating Disk → Pelham's Hovering Platter
+> - Hideous Laughter → Paoliello's Hideous Laughter
+> - Magnificent Mansion → Alia's Magnificent Mansion
+> - Secret Chest → Cerilsen's Secret Chest
+> - Telepathic Bond → Rantock's Telepathic Bond
+> - Tiny Hut → Melestrua's Marvelous Marquee
```

Comment:
This section is inspired by the information about renaming classes.
It will probably be useful for people playing ready made adventures, since they can look up the new name of the spell.

What I have written is just a suggestion for what it could contain and how it could be worded of course; I'm open to alternative suggestions.